WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -504,9 +504,13 @@ def execute( self ):
     self.cfg.append(
         '-o "/Resources/Computing/CEDefaults/NumberOfProcessors=%d"' % numberOfProcessors)
 
-    maxRAM = self.pp.queueParameters.get('MaxRAM', maxRAM)
+    maxRAM = int(self.pp.queueParameters.get('MaxRAM', maxRAM))
```

Comment:
the cast to `int` should be dropped here, as it is done again below and there with try/except